string command
string[] args
---
int16 status
string message
